src/liger_kernel/transformers/embedding.py

```diff
@@ -0,0 +1,28 @@
+from typing import Optional
+
+import torch
+import torch.nn as nn
+
+from liger_kernel.ops.embedding import LigerEmbeddingFunction
+
+
+class LigerEmbedding(nn.Module):
+    def __init__(
+        self, num_embeddings, embedding_dim, padding_idx: Optional[int] = None
+    ):
+        super().__init__()
```

Comment:
We can merge it for now, but add a warning here say this is still not fully optimized

Reply:
actually, let's create `src/liger_kernel/ops/experimental`, just like https://github.com/google/jax/tree/main/jax/experimental/mosaic

Reply:
okay, I'll make that change quickly

Reply:
made changes @ByronHsu